modules/xfem/src/userobjects/CrackMeshCut3DUserObject.C

```diff
@@ -831,29 +840,31 @@ CrackMeshCut3DUserObject::growFront()
       Point dir = _active_direction[i][j];
 
       Point x;
-
-      if (_growth_rate_method == GrowthRateEnum::FUNCTION)
-        for (unsigned int k = 0; k < 3; ++k)
+      Real growth_size = 0;
```

Comment:
With the rename from growth_size to growth_increment, go through and rename all related variables like this one to _increment for consistency.